KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift

```diff
@@ -4,26 +4,64 @@ import ReactiveCocoa
 import Kiosk
 import Nimble_Snapshots
 
+class PlaceBidViewControllerConfiguration: QuickConfiguration {
+    override class func configure(configuration: Configuration) {
+        sharedExamples("a bid view controller view controller", { (sharedExampleContext: SharedExampleContext) in
+            var sut: PlaceBidViewController!
+            var nav: FulfillmentNavigationController!
+
+            beforeEach {
+                sut = sharedExampleContext()["sut"] as PlaceBidViewController!
+                nav = sharedExampleContext()["nav"] as FulfillmentNavigationController!
+            }
+
+            describe("with lot number") {
+                beforeEach {
+                    nav.bidDetails.saleArtwork?.lotNumber = 13
+                    return
```

Comment:
Why are you using an explicit `return` here? (More of Swift question, I assume.)